File: tests/multilang_rule_coverage.go
package main

import (
"crypto/md5"
"crypto/sha1"
"crypto/tls"
"os/exec"
)

func main() {
userInput := "echo test"
exec.Command("sh", "-c", userInput)
_ = md5.New()
_ = sha1.New()
_ = &tls.Config{InsecureSkipVerify: true}
}

File: tests/multilang_rule_coverage.rs
use std::process::Command;

fn main() {
    let user_input = "echo test";
    let data = b"secret";
    let _ = Command::new("sh").arg("-c").arg(user_input).status();
    let _ = md5::compute(data);
    let _ = reqwest::Client::builder().danger_accept_invalid_certs(true).build();
}
